app/main/posts/posts-routes.js

```diff
@@ -121,10 +121,6 @@ function (
             name: 'posts.map.all',
             views: {
                 'mode-context': 'modeContext'
-            },
-            onEnter: function ($state, $transition$, PostFilters, savedSearch) {
-                PostFilters.setMode('all');
```

Comment:
I suspect setMode still belongs here. In fact from memory that has some logic that is mode changes it resets the filters right?
But resetDefaults should go.